components/analytics/components/SearchLatency/index.js

```diff
@@ -132,6 +131,7 @@ SearchLatency.propTypes = {
 	searchLatency: PropTypes.array.isRequired,
 	isLoading: PropTypes.bool.isRequired,
 	success: PropTypes.bool.isRequired,
+	selectFilterValue: PropTypes.func.isRequired,
```

Comment:
@jyash97 Why this prop is required?
`Filters` are a higher plan feature.

Reply:
This action here is just used to apply Filter Params. Check this: https://github.com/appbaseio/batteries/pull/220/files#diff-ec9e740f6133c59a3f9717c822082b7eR28